docs/sdks/js-reference-guide/js-sdk-redeemOtp.md

```diff
@@ -0,0 +1,91 @@
+---
+title: redeemOtp
+id: js-reference-redeemOtp
+description: ""
+slug: /js-reference-redeemOtp
+keywords:
+  - javascript sdk
+pagination_next: null
+pagination_prev: null
+last_update:
+  date: 07/25/2023
+  author: William May
+draft: false
+doc_type: reference
+displayed_sidebar: sdkSidebar
+---
+
+The **redeemOtp** function enables an app using the Beyond Identity Javascript SDK to redeem an otp for a grant code. This function is used in conjunction with [authenticateOtp](js-reference-authenticateOtp).
+
+## Dependencies
+
+The **redeemOtp** function requires the Beyond Identity Javascript SDK.
+
+```
+npm install @beyondidentity/bi-sdk-js
+```
+
+## Prerequisites
+
+Before making a call to **redeemOtp**, you must complete the following prerequisite calls:
+
+1. Import the required types and functions from the SDK
+
+```javascript
+import { Embedded } from "@beyondidentity/bi-sdk-js";
+```
+
+2. Initialize the SDK
+
+```javascript
+const embedded = await Embedded.initialize();
+```
+
+3. Use **redeemOtp** to redeem an otp for a grant code
+
+```javascript
+await embedded.redeemOtp(url, otp);
+```
```

Comment:
I wonder if authenticatiOtp should be a prerequisite here too

Reply:
Added.